tests/AgentTests/FunctionalAPITests.hs

```diff
@@ -597,7 +599,7 @@ runAgentClientTestPQ :: HasCallStack => Bool -> (AgentClient, InitialKeys) -> (A
 runAgentClientTestPQ viaProxy (alice, aPQ) (bob, bPQ) baseId =
   runRight_ $ do
     (bobId, qInfo) <- A.createConnection alice 1 True SCMInvitation Nothing aPQ SMSubscribe
-    aliceId <- A.joinConnection bob 1 Nothing True qInfo "bob's connInfo" bPQ SMSubscribe
+    (aliceId, _sqSecured) <- A.joinConnection bob 1 Nothing True qInfo "bob's connInfo" bPQ SMSubscribe
```

Comment:
looks like you either need joinConnection' here , or, better, a proper pattern match that it returns expected value here, so it's tested.